G1ANT.Appium/Commands/AppiumScrollCommand.cs

```diff
@@ -13,12 +13,12 @@ public class ScrollCommand : Language.Command
     {
         public class Arguments : CommandArguments
         {
-            [Argument(Tooltip = "Specify by wich capability the element will be searched")]
-            public TextStructure By { get; set; } = new TextStructure("");
-
             [Argument(Tooltip = "Provide name of the capaility")]
```

Comment:
Same arguments as in "G1ANT.Appium/Commands/PresentCommand.cs → G1ANT.Appium/Commands/AppiumPresentCommand.cs"

Extract them to a base class.